test/module/options/normalizeUrlSpec.js

```diff
@@ -0,0 +1,26 @@
+import 'source-map-support/register';
```

Comment:
I wasn't sure myself but I left it in whilst I wasn't fully aware what was needed to run tests with the existing setup. I've removed it and all seems fine so I can take this out